v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -348,20 +348,20 @@ protected static java.lang.Class indexTypeToColumnClass(SourceColumnIndexInfo in
   }
 
   /**
-   * Delimit the Column Names as per <a
+   * Delimit the Identifiers as per <a
    * href=https://github.com/ronsavage/SQL/blob/master/sql-99.bnf>sql-99</a>.
    *
-   * @param columnName
+   * @param identifier
```

Comment:
Coudl you please add some details on why this is needed, not obvious from the link

Reply:
Added that we need this to handle reserved keywords as column or table names.